hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -1600,6 +1600,24 @@
     </description>
   </property>
 
+  <property>
+    <name>ozone.om.unflushed.transactions.max.size</name>
```

Comment:
Can we change the "transactions" to "transaction"?

Reply:
Also, I think "count" would better reflect the meaning of this config than "size".